pyenvbuilder/commands/create.py

```diff
@@ -123,9 +122,11 @@ def conda_create(self, data):
         tests_template = Template(
             'echo \n\n'
             'echo ------ TESTING ----- \n'
+            'pushd $source_dir\n'
```

Comment:
I am not to sure about this.... I took it from your scripts Hugo... please let me know if I should keep this or do something else here. 
Basically this should fix the problem with creating an environment with an `env.yml` file from a different location.
This problem is independent of the `--dest` option - not related

Reply:
Most likely my script was also not working properly 😅

Reply:
I think your script was working fine. If I don't `pushd` to the directory where the tests are, the alternative would be to parse the tests list, and get the full path of for each test... but that will get messy

Reply:
Correct, we must switch to the directory but I guess my order was wrong... we must first activate the environment (which is at the cwd) and after that change to the yaml file directory which will be parallel to tests. 👍

Reply:
So we should have something like this: 
```
            'echo ------ TESTING ----- \n'
            'source $activate_script\n'
            'conda activate $env_path\n'
            'pushd $source_dir > /dev/null\n'
            '$tests\n'
            'popd > /dev/null\n')
```
Is that what you mean?

They both worked for me.... I don't know if that is a good thing hahah

Reply:
Most likely they both worked because the path to environment was the full path... in this case any situation is fine! 👍

Reply:
Ohh ok got it, I will leave it like this then, thank you